### PR TITLE
pylint: add page

### PR DESCRIPTION
**Pull request by adamatan**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
pylint 2.11.1 on Python 3.10

### CODE REVIEW

File: pages/common/pylint.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# pylint
+
+> A Python code linter.
+> More information: <https://pylint.pycqa.org/en/latest/>.
+- Show lint errors in a file:
```
Comment (marchersimon) [inline pages/common/pylint.md:5]: This fixes the error
```suggestion

- Show lint errors in a file:
```

File: pages/common/pylint.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`pylint file`
```
Comment (marchersimon) [inline pages/common/pylint.md:7]:
```suggestion
`pylint {{path/to/file.py}}`
```

File: pages/common/pylint.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- Use the configuration from a configuration file, usually named `pylintrc`:
```
Comment (marchersimon) [inline pages/common/pylint.md:9]:
```suggestion
- Lint a file and use a configuration file (usually named `pylintrc`):
```

File: pages/common/pylint.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+`pylint --rcfile ~/path/to/pylintrc file`
```
Comment (marchersimon) [inline pages/common/pylint.md:11]:
```suggestion
`pylint --rcfile {{path/to/pylintrc}} {{path/to/file.py}}`
```

File: pages/common/pylint.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+- Disable a specific error code:
```
Comment (marchersimon) [inline pages/common/pylint.md:13]:
```suggestion
- Lint a file and disable a specific error code:
```

File: pages/common/pylint.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+`pylint -d error_code file`
```
Comment (marchersimon) [inline pages/common/pylint.md:15]: I think we should give an example here
```suggestion
`pylint --disable {{C,W,no-error,design}} {{path/to/file}}`
```

File: pages/common/pylint.md (hunk 3 above)
Comment (adamatan) [reply pages/common/pylint.md:10]: Great suggestion!

File: pages/common/pylint.md (hunk 2 above)
Comment (adamatan) [reply pages/common/pylint.md:8]: Good point.

File: pages/common/pylint.md (hunk 6 above)
Comment (adamatan) [reply pages/common/pylint.md:16]: Totally agree, it's much clearer that way.

**Comment by adamatan** [comment]
Thanks a lot for the review! It's way better now.

**Review by marchersimon — APPROVED** [review]
Thanks @adamatan

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**